c-sharp/c-sharp-quiz.md

```diff
@@ -793,3 +793,10 @@ void MyFunction()
 - [x] Variable c no longer exists outside the block.
 
 [Reference](https://stackoverflow.com/a/6136865)
+
+    #### Q74. Which statement is True?
```

Comment:
Let's format it a bit

```suggestion
#### Q74. Which statement is True?
```

Reply:
going to merge but please do improvements regarding this comment later, thanks